command/search_test.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright (C) 2018 Expedia Group.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"github.com/HotelsDotCom/flyte-jira/client"
+	"net/http"
+	"reflect"
+	"testing"
+)
+
+func TestSearchIssuesSuccess(t *testing.T) {
+	client.SendRequest = func(request *http.Request, responseBody interface{}) (int, error) {
```

Comment:
We should use defer to clean up `SendRequest` replacement. 

```
oldSendRequest := client.SendRequest
client.SendRequest = func(request *http.Request, responseBody interface{}) (int, error) {
...
}
defer func() { client.SendRequest = oldSendRequest}()
```

Reply:
Refactored all of the test functions to this